### PR TITLE
fix: should now work with newer versions of React Native + expo

### PR DESCRIPTION
**Pull request by tormodAase**

### CODE REVIEW

**Comment by tormodAase** [comment]
@HaakonSvane since you're our xcode expert I added you as reviewer as well.

@HaakonSvane @HavardNot good luck soldiers

File: android/src/main/java/com/zebralinkos/ZebraLinkOsModule.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    return NAME;
+  }
+
+
+  // Example method
+  // See https://reactnative.dev/docs/native-modules-android
+  @ReactMethod
+  public void multiply(double a, double b, Promise promise) {
+    promise.resolve(a * b);
+  }
```
Comment (HavardNot) [inline android/src/main/java/com/zebralinkos/ZebraLinkOsModule.java:31]: should be removed?

File: android/src/main/java/com/zebralinkos/ZebraLinkOsModule.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ReactMethod;
+import com.facebook.react.module.annotations.ReactModule;
+
+@ReactModule(name = ZebraLinkOsModule.NAME)
+public class ZebraLinkOsModule extends ReactContextBaseJavaModule {
+  public static final String NAME = "ZebraLinkOs";
+
+  public ZebraLinkOsModule(ReactApplicationContext reactContext) {
```
Comment (HaakonSvane) [inline android/src/main/java/com/zebralinkos/ZebraLinkOsModule.java:15]: Is this (or any other android) file intended?

File: example/ios/File.swift [hunk 3]
```diff
@@ -0,0 +1,6 @@
+//
+//  File.swift
```
Comment (HaakonSvane) [inline example/ios/File.swift:2]: Is this file intended?

**Review by HaakonSvane — APPROVED** [review]
Hella fresh 🔥 Looking forward to printing out pictures of birds with this 🐦

**Review by HavardNot — APPROVED** [review]
🦓

File: android/src/main/java/com/zebralinkos/ZebraLinkOsModule.java (hunk 1 above)
Comment (tormodAase) [reply android/src/main/java/com/zebralinkos/ZebraLinkOsModule.java:31]: Removed everything related to Android

File: example/ios/File.swift (hunk 3 above)
Comment (tormodAase) [reply example/ios/File.swift:2]: File.swift was autogenerated, including the comment at the top

File: android/src/main/java/com/zebralinkos/ZebraLinkOsModule.java (hunk 2 above)
Comment (tormodAase) [reply android/src/main/java/com/zebralinkos/ZebraLinkOsModule.java]: Removed everything related to Android

File: example/src/App.tsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+
+import { StyleSheet, View, Text } from 'react-native';
+import RNZebraLinkOs from 'react-native-zebra-link-os';
+
+export default function App() {
+  console.log(RNZebraLinkOs);
```
[the remark this answers is not in the document]
Comment (tormodAase) [reply example/src/App.tsx:8]: This console.log is the only thing the example app does. It checks if RNZebraLinkOs exists.
Added a comment to reflect that.

[the remark this answers is not in the document]
Comment (tormodAase) [reply example/src/App.tsx:8]: This console.log is the only thing the example app does. It checks if RNZebraLinkOs exists.
Added a comment to reflect that.
